packages/admin/test/e2e/cli.feature.ts

```diff
@@ -41,7 +41,7 @@ describe('CLI', () => {
     if (DEBUG_COMMANDS) console.log(`Executing command: ${formattedCommand}`);
     try {
       return execSync(`node ${CLI_EXECUTABLE} ${formattedCommand}`).toString().trim();
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
I'd leave it as any. There is not a native TS type for this, and creating one seems unnecessary since this is just for a single test anyway...